Create ui-c8y-1023-81-4-improve-ng-add-instructions-with-custom-web-sdk-instructions.md (#4723)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-81-4-improve-ng-add-instructions-with-custom-web-sdk-instructions.md

---
date: ""

> title: Improve ng add instructions with custom Web SDK instructions

Reviewer: ```suggestion
title: Custom Web SDK coding agent instructions adding during app creation
```

> title: Custom Web SDK coding agent instructions adding during app creation

Reviewer 2: ```suggestion
title: Custom Web SDK coding agent instructions added during app creation
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66393
version: 1023.81.4
---

> User creating his app with @c8y/websdk 'ng add' command will now be prompted with list of possible coding agents to add custom instructions during the process. These instructions contains context of the project, good practices, common ways of implemeneting things, points out where to find resources related to @c8y packages.

Reviewer: ```suggestion
When you create an app using the @c8y/websdk 'ng add' command, you are now prompted with a list of coding agents. Once selected, custom instructions tailored to your project will be added in format that matches agents of your choice. These instructions include project context, best practices, common implementation patterns, and guidance on where to find resources related to @c8y packages, helping you get started more quickly and follow recommended approaches from the start.
```

> When you create an app using the @c8y/websdk 'ng add' command, you are now prompted with a list of coding agents. Once selected, custom instructions tailored to your project will be added in format that matches agents of your choice. These instructions include project context, best practices, common implementation patterns, and guidance on where to find resources related to @c8y packages, helping you get started more quickly and follow recommended approaches from the start.

Reviewer 2: ```suggestion
When creating an app using the @c8y/websdk `ng add` command, you are now prompted to select coding agents from a list. Once selected, custom instructions tailored to your project are added in a format that matches the agents of your choice. These instructions include project context, best practices, common implementation patterns, and guidance on where to find resources related to @c8y packages, helping you to get started more quickly and follow recommended approaches from the start.
```

> When creating an app using the @c8y/websdk `ng add` command, you are now prompted to select coding agents from a list. Once selected, custom instructions tailored to your project are added in a format that matches the agents of your choice. These instructions include project context, best practices, common implementation patterns, and guidance on where to find resources related to @c8y packages, helping you to get started more quickly and follow recommended approaches from the start.

Reviewer 2: @jdre-c8y Do I select one or multiple agents?

Reviewer: you can select none, one or multiple- if multiple are selected, each of selected will get instructions suited for agent (same content, but different format)

Reviewer 2: ok, then the text is fine.
